installation/product-opener.md

```diff
@@ -0,0 +1,7 @@
+# Product Opener Installation
+
+Product opener is released under the AGPL license and is being developed as [Free and Open Source](https://en.wikipedia.org/wiki/Free_and_open-source_software)  on Github.
+
+### Installation
+Installation Instructions for Product Opener
+[Product Opener Installation](https://en.wiki.openfoodfacts.org/Product_Opener/Installation)
```

Comment:
This wiki page is very old and contained outdated installation instructions.

I replaced its content with a link to https://en.wiki.openfoodfacts.org/Infrastructure#Server_configuration which contains newer instructions (but not necessarily completely up to date either).